### PR TITLE
Fix #101: vagrant-service-manager version 0.0.4 release

### LINKED ISSUE #101: vagrant-service-manager version 0.0.4 release
Tracker issue for 0.0.4
- Target issues for the release:
  - Fixed:
    - #95 
    - #94 
    - #91 
    - #72 
    - #97 
    - PR https://github.com/projectatomic/vagrant-service-manager/pull/107

### PR DESCRIPTION
**Pull request by navidshaikh**
Fix #101

### CODE REVIEW

**Comment by navidshaikh** [comment]
Do not merge unless we have #88 merged.

**Comment by bexelbie** [comment]
It'd be nice if #107 makes too

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-## Unreleased
+## v0.0.4 Mar 14, 2016
+- Fix #101: vagrant-service-manager version 0.0.4 release @navidshaikh
+- Fix #87 #83: Supports starting OpenShift service as part of config @budhrg @bexelbie @navidshaikh
+- Fix #95: Update hook code to call other middleware first @bexelbie
 - Fix #94: Do not exit if box is not supported @navidshaikh
 - Fixed missing word for plugin installation in README @budhrg
 - Fix links, typos, formatting in CONTRIBUTING.md @budhrg
```
Comment (LalatenduMohanty) [inline CHANGELOG.md]: os_variant patch aslo should be here. I did not update this with the original PR

Comment (LalatenduMohanty) [inline CHANGELOG.md:11]: os_variant patch also should be here. I did not update this with the original PR

**Comment by LalatenduMohanty** [comment]
@bexelbie @navidshaikh We should release asap as it has fix for blocker bugs

File: CHANGELOG.md (hunk 1 above)
Comment (navidshaikh) [inline CHANGELOG.md:11]: @LalatenduMohanty : I apologize, I should have cross checked it earlier. Will update the PR

**Comment by bexelbie** [comment]
LGTM

**Pull request merged**